src/pages/patterns/lead-space.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Lead space
+description: A Lead space is positioned at the top of a web page and serves as the first site of viewer engagement. Its primary purpose is to orient the user when they land on a page, inform them of the content, and guide them to the first key piece of content on the page, whether it’s a crucial piece of information or a call-to-action button.
+  
+---
+
+<PageDescription>
+
+A Lead space is positioned at the top of a web page and serves as the first site of viewer engagement. Its primary purpose is to orient the user when they land on a page, inform them of the content, and guide them to the first key piece of content on the page, whether it’s a crucial piece of information or a call-to-action button.
+
+When designed and implemented effectively, the lead space can be used to engage visitors immediately in one of the most important moments in the user’s journey through a complex web environment. It is the first section below the masthead and can include a stylized background, headline, supporting message and call to action to accommodate user needs.
+
+</PageDescription>
+
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Dimensions</AnchorLink>
+<AnchorLink>Size standards</AnchorLink>
+<AnchorLink>Content alignment</AnchorLink>
+<AnchorLink>Related components and patterns</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+A compelling lead space can encourage customers to spend time on your website. Depending on the needs of your user and the information you are trying to share, the lead space of your webpage may take different forms.
+
+## Dimensions
+
+Lead spaces are used to enhance the content or introduce a supporting narrative. There are multiple dimensions to consider when designing a Lead space. 
+
+The dimensions for consideration include:
+
+#### Journey segments
+Where is the page you are creating in the user journey? Is it in the early, mid, checkout, or post-purchase?
+
+#### Variations within segments per page type
+Is the page you are creating for product listing? if not, is it for a business solution or a landing page, etc?
+
+#### Expressive VS productive
+Is the page you are creating used for presenting typographical content for users to gain insight or for helping users to focus on a task?
+
+#### Centered VS left-aligned
+No matter which segments your offers are in the user journey, always make sure you have a clear understanding of the purpose of the page. See the Content alignment section below for detail.
+
+We are always refining and testing the standards, so please contribute by working with us to refine the standard details.
+
+## Size standards
+
+The curated Lead Space size standards are super, tall, regular, short, mini. please start by using the sizes listed here so we can create consistency for the IBM.com experience.
```

Comment:
Let's add some messages around using the right size for the right purpose. My suggestion:

"The curated Lead Space size standards are super, tall, regular, short, mini. Choose the right size for the purpose of the lead space on the given page."

Reply:
Sounds good. FYI: We use Lead space in the sentence when calling out a component or pattern. Let me know if there are any changes to that rule I am not aware of.